src/helper/bitmap.js

```diff
@@ -86,17 +86,19 @@ const fillEllipse = function (centerX, centerY, radiusX, radiusY, context) {
 /**
  * @param {!number} size The diameter of the brush
  * @param {!string} color The css color of the brush
+ * @param {?boolean} isEraser True if we want the brush mark for the eraser
  * @return {HTMLCanvasElement} a canvas with the brush mark printed on it
  */
-const getBrushMark = function (size, color) {
+const getBrushMark = function (size, color, isEraser) {
```

Comment:
seems like the consumer of this could just pass in `white` for the color if they wanted that? Could you just change brush-tool to use `getBrushMark(this.size, this.isEraser ? 'white' : this.color)`?

Reply:
Oh nevermind I see the comment about adding the outline as well

Reply:
I could, but I want to draw an outline around the white circle for erasers, just as soon as I get the ellipse code in...